File: app/src/main/java/employee/summon/asano/model/Person.java
package employee.summon.asano.model;

public class Person extends PersonBase {
    private Integer id;

    public Person(Integer id, String firstName, String lastName, String email, String password, Integer departmentId, boolean online) {
        super(firstName, lastName, email, password, departmentId, online);
        this.id = id;
    }

    public Integer getId() {
        return id;
    }
}

File: app/src/main/java/employee/summon/asano/model/Person.kt
package employee.summon.asano.model

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

@Parcelize
class Person(val id: Int, val firstname: String, val lastname: String, val patronymic: String?,
             val post: String?, val email: String?, val phone: String?) : Parcelable
